shared/components/project-cards.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from "react";
+import { useThemeHooks } from '@redocly/theme/core/hooks';
+
+interface Project {
+  id: string;
+  label: string;
+  url: string;
+  description?: string; // New optional field for payments page
+  buttonText?: string; // Optional button text for battle-tested cards
+}
+
+interface ProjectCardsProps {
+  title: string;
+  projects: Project[];
+  showCarousel?: boolean; // true for tokenization (carousel), false for payments (grid)
+  className?: string;
+}
+
+const ProjectCard = ({ project, index, showCarousel = true }: { 
+  project: Project; 
+  index: number; 
+  showCarousel?: boolean;
+}) => {
+  const { useTranslate } = useThemeHooks();
+  const { translate } = useTranslate();
+
+  return (
+    <a
+      className={`col card float-up-on-hover ${
+        showCarousel 
+          ? (index % 2 === 0 ? "even" : "odd") 
+          : `payments-project-card ${index % 2 === 0 ? "odd" : "even"}`
+      }`}
+      target="_blank"
+      href={project.url}
+    >
+      <div className="project-logo d-flex justify-content-center align-items-center">
+        <img className={project.id} alt={project.label} />
+      </div>
+      {!showCarousel && project.description && (
+        <div className="project-description">
+          {(() => {
+            const words = project.description.split(' ');
+            const firstWord = words[0];
+            const restOfText = words.slice(1).join(' ');
+            return (
+              <>
+                <strong className="first-word">{firstWord}</strong>
+                {restOfText && <span className="rest-text"> {restOfText}</span>}
+              </>
+            );
+          })()}
+        </div>
+      )}
+      {!showCarousel && project.buttonText && (
+        <div className="project-button">
+          <a href={project.url} className="btn-arrow battle-tested-arrow">{translate(project.buttonText)}</a>
```

Comment:
This is invalid markup and is causing a hydration error because this `<a>` element is a descendant of the card which is also an `<a>` element (started on line 28). Changing it to a span fixes the problem and doesn't affect it visually.

```suggestion
          <span className="btn-arrow battle-tested-arrow">{translate(project.buttonText)}</span>
```